components/lcd_touch/esp_lcd_touch_stmpe610/include/esp_lcd_touch_stmpe610.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include "esp_lcd_touch.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Create a new STMPE610 touch driver
+ *
+ * @note The SPI communication should be initialized before use this function.
+ *
+ * @param io LCD/Touch panel IO handle
+ * @param config: Touch configuration
+ * @param out_touch: Touch instance handle
+ * @return
+ *      - ESP_OK                    on success
+ *      - ESP_ERR_NO_MEM            if there is no memory for allocating main structure
+ *      - ESP_ERR_INVALID_ARG       if there is used bad I2C number or GPIO
```

Comment:
SPI?

Reply:
Same as before on I2C --> removed this comment.